tracer/test/test-applications/integrations/Samples.AWS.DynamoDBv2/AsyncHelpers.cs

```diff
@@ -118,7 +118,7 @@ public static async Task BatchWriteItemAsync(AmazonDynamoDBClient dynamoDBClient
             };
             var batchWriteItemRequest = new BatchWriteItemRequest
             {
-                RequestItems =
+                RequestItems = new()
```

Comment:
These collections are `null` by default now, so need to explicitly new them up (build warning advised about this, and throws a null ref at runtime otherwise)